### ISSUE TITLE
Comunidade fora do Brasil - Portugal

### ISSUE THREAD

**Issue by gabriela-castro** [training-center/forum#40]
Olá a todos! Tem alguém morando/trabalhando fora do Brasil, principalmente em Portugal? 
Talvez possamos nos organizar em um grupo para trocar idéias, dicas, experiências.

**Comment by lasombra** [comment]
Gostei da ideia. Podemos começar com uma lista dos países onde cada um está e ir expandindo daí. O que você acha?

**Comment by gabriela-castro** [comment]
Boa ideia! Vamos esperar as pessoas participarem e nós fazemos a lista.

**Comment by WendellAdriel** [comment]
Moro em Lisboa/Portugal desde 2016!!! :wink:

**Comment by rodrigobravo** [comment]
Estou em processo de mudança para Portugal nos próximos meses, apesar de ainda estar no Brasil, posso acompanhar a discussão ?

**Comment by lasombra** [comment]
Moro no sul da Inglaterra desde 2013.

**Comment by woliveiras** [comment]
OLHAE! TÁ VINDO UM MEETUP DO TRAINING CENTER PRA BRs NAS EUROPA ENTÃO?

**Comment by lucasbento** [comment]
Gostei bastante da ideia, moro em Amsterdam, Holanda!

**Comment by ghost** [comment]
Showw! Moro em Hannover na Alemanha.

**Comment by clucasalcantara** [comment]
Opa! Também adorei a ideia! To em Lisboa  :)

**Comment by clucasalcantara** [comment]
Então pessoal, vamos animar isso? 

Eu consigo correr atrás do espaço pro meetup e tal, se quiserem falar sobre JS eu tb to disponivel :)

**Comment by nicolastakashi** [comment]
Estou em Braga, desde 2017

**Comment by gabriela-castro** [comment]
Bora sim! Até agora temos:

- Alemanha : 1
- Holanda : 1
- Inglaterra : 1
- Portugal : 4

Onde @clucasalcantara, você tem como ver o espaço? E sobre o que vocês gostariam de falar? 
Acho que devemos falar sobre alguma experiencia de emprego  na Europa e sobre algum trabalho/linguagem muito comum aqui.

**Comment by WendellAdriel** [comment]
Eu estou cá em Portugal desde 2016. Moro em Lisboa. Se toparem eu posso ficar como um dos responsáveis pelo meetup TC Portugal. Como devemos prosseguir @woliveiras???

**Comment by woliveiras** [comment]
Assim que chegarem no acordo aqui, é só criar seguindo este guideline, Mano

https://github.com/training-center/meetups

Nesse doc, só a parte do meetup.com que está em discussão pra reduzir custos

**Comment by clucasalcantara** [comment]
Vou procurar uns offices maneiros que disponibilizam espaço
pra galera e retorno aqui!

**Comment by williangd** [comment]
Sobre Portugal, como que são as empresas por aí? Tem bastante vaga?

**Comment by WendellAdriel** [comment]
@williangd creio que aqui não seja o local mais adequado para discutir esses assuntos para não fugirmos do foco.
Se quiser me procurar no Slack do TC: `wendell_adriel` posso esclarecer algumas dúvidas!!! :wink:

**Comment by woliveiras** [comment]
Acho mais válido falar em outro tópico do que no privado, viu mano @WendellAdriel.

Assim as suas respostas vão ajudar outras pessoas.

Pode abrir uma issue com essa questão, @williangd?

**Comment by kivervinicius** [comment]
@williangd  @WendellAdriel  gostaria de saber também!

**Comment by WendellAdriel** [comment]
Sucesso @woliveiras, abram uma issue sobre e me marquem por lá!!! :wink:

**Comment by tiagocesar** [comment]
@WendellAdriel bom te ver por aqui :-)

**Comment by tiagocesar** [comment]
+1 pra Amsterdam

**Comment by WendellAdriel** [comment]
@tiagocesar participo do TC desde quando foi criado né @woliveiras ???
HAAHAHAHHHAHA

Bom saber que está por aqui também!!!

**Comment by WendellAdriel** [comment]
@woliveiras eu conversei com uma empresa cá de Lisboa para fazer parceria e eles sediarem espaço para o Meetup do TC e vão me dar resposta na semana que vem. Qualquer novidade aviso!!!

**Comment by clucasalcantara** [comment]
Seria a Hexis? :p

**Comment by WendellAdriel** [comment]
Sim... e fiquei sabendo ha pouco que você é Tech Lead lá @clucasalcantara xD

**Comment by clucasalcantara** [comment]
:D só vamo mano! Vamos marcar pra gente se bater e organizar isso direito
assim que o @woliveiras autorizar a gente fazer essa bagaça hehe

**Comment by WendellAdriel** [comment]
Assim que o pessoal aí da empresa tiver um feedback me avisa que vou aí pra gente trocar umas ideias!!! Depois é só seguir o passo-a-passo do repo de modelo dos meetups e ficamos nós dois responsáveis por ele por enquanto e depois achamos mais gente pra ajudar a gente!!!

**Comment by clucasalcantara** [comment]
Show! O Matheus deve colar pra ajudar a gente tb :)

Então, falei com o pessoal de operações hoje e na segunda já pego o aval do nosso CEO, a gente deve tá indo pro nosso office na segunda semana de fevereiro, passando esse gap de levar e montar as coisas já da pra gente marcar uma data.

Espera tudo isso não! cola lá! vai almoçar com a gente :P

**Comment by WendellAdriel** [comment]
Opa massa demais!!! O Mateus ajudando vai ser melhor ainda!!!

A Rafa me falou que já vão mudar de office!!! Almoço fica maia difícil porque meu trampo é longe daí, mas se quiserem marcar algo depois do trabalho é na hora!!! xD

**Comment by woliveiras** [comment]
Fala, galera!

Não precisa nem de permissão.

A comunidade é nossa! \o/

Podemos criar um https://www.meetup.com/pt-BR/Training-Center-LISBON então?

**Comment by WendellAdriel** [comment]
@woliveiras já criei e enviei o PR

**Comment by woliveiras** [comment]
Tá lá! \o/

bit.ly/tc-meetups

**Comment by Morette** [comment]
Bom dia pessoal, aproveitando essa issue.
Como que é o mercado de TI por Portugal para quem não tem experiência de mercado ainda? Há cargos de Jr. disponível?

Pretendo me mudar para Portugal (talvez ainda este ano), porém ainda estou estudando.

**Comment by wesmelo** [comment]
Boa @Morette, também tenho interesse em saber como que é o mercado de TI por Portugal para quem está começando.

**Comment by WendellAdriel** [comment]
@Morette e @melowesley vejam essa issue: https://github.com/training-center/forum/issues/44
